### ISSUE TITLE
Transition error in CoolingTower:SingleSpeed

### ISSUE THREAD

**Issue by macumber** [NREL/EnergyPlus#6063]
We found an error in the 8.7 prerelease version translation for one of our test files.

**Comment by macumber** [comment]
This is the error we got when running the updated file:

```
** Severe  ** Out of range value Numeric Field#13 (Design Inlet Air Dry-Bulb Temperature), value=1.25000, range={>=20.0}, in COOLINGTOWER:SINGLESPEED=COOLING TOWER SINGLE SPEED 1
** Severe  ** Out of range value Numeric Field#15 (Design Approach Temperature), value=0.00000, range={>0}, in COOLINGTOWER:SINGLESPEED=COOLING TOWER SINGLE SPEED 1
```

Here is the object in E+ 8.6:
```
CoolingTower:SingleSpeed,
    Cooling Tower Single Speed 1,  !- Name
    Node 148,                !- Water Inlet Node Name
    Node 152,                !- Water Outlet Node Name
    Autosize,                !- Design Water Flow Rate {m3/s}
    Autosize,                !- Design Air Flow Rate {m3/s}
    Autosize,                !- Design Fan Power {W}
    Autosize,                !- Design U-Factor Times Area Value {W/K}
    Autosize,                !- Free Convection Air Flow Rate {m3/s}
    0.1,                     !- Free Convection Air Flow Rate Sizing Factor
    Autosize,                !- Free Convection U-Factor Times Area Value {W/K}
    0.1,                     !- Free Convection U-Factor Times Area Value Sizing Factor
    UFactorTimesAreaAndDesignWaterFlowRate,  !- Performance Input Method
    1.25,                    !- Heat Rejection Capacity and Nominal Capacity Sizing Ratio
    ,                        !- Nominal Capacity {W}
    0,                       !- Free Convection Capacity {W}
    0.1,                     !- Free Convection Nominal Capacity Sizing Factor
    0,                       !- Basin Heater Capacity {W/K}
    2,                       !- Basin Heater Setpoint Temperature {C}
    ,                        !- Basin Heater Operating Schedule Name
    ,                        !- Evaporation Loss Mode
    0.2,                     !- Evaporation Loss Factor {percent/K}
    0.008,                   !- Drift Loss Percent {percent}
    ,                        !- Blowdown Calculation Mode
    3,                       !- Blowdown Concentration Ratio
    ,                        !- Blowdown Makeup Water Usage Schedule Name
    ,                        !- Supply Water Storage Tank Name
    ,                        !- Outdoor Air Inlet Node Name
    FanCycling,              !- Capacity Control
    1,                       !- Number of Cells
    MinimalCell,             !- Cell Control
    0.33,                    !- Cell Minimum  Water Flow Rate Fraction
    2.5,                     !- Cell Maximum Water Flow Rate Fraction
    1;                       !- Sizing Factor
```
    
Here is the object after transition to E+ 8.7:
```
CoolingTower:SingleSpeed,
    Cooling Tower Single Speed 1,  !- Name
    Node 148,                !- Water Inlet Node Name
    Node 152,                !- Water Outlet Node Name
    Autosize,                !- Design Water Flow Rate {m3/s}
    Autosize,                !- Design Air Flow Rate {m3/s}
    Autosize,                !- Design Fan Power {W}
    Autosize,                !- Design U-Factor Times Area Value {W/K}
    Autosize,                !- Free Convection Air Flow Rate {m3/s}
    0.1,                     !- Free Convection Air Flow Rate Sizing Factor
    Autosize,                !- Free Convection U-Factor Times Area Value {W/K}
    0.1,                     !- Free Convection U-Factor Times Area Value Sizing Factor
    UFactorTimesAreaAndDesignWaterFlowRate,  !- Performance Input Method
    ,                        !- Heat Rejection Capacity and Nominal Capacity Sizing Ratio
    ,                        !- Nominal Capacity {W}
    ,                        !- Free Convection Capacity {W}
    ,                        !- Free Convection Nominal Capacity Sizing Factor
    1.25,                    !- ***** Design Inlet Air Dry-Bulb Temperature {C} *****
    ,                        !- ***** Design Inlet Air Wet-Bulb Temperature {C} *****
    0,                       !- Design Approach Temperature {deltaC}
    0.1,                     !- Design Range Temperature {deltaC}
    0,                       !- Basin Heater Capacity {W/K}
    2,                       !- Basin Heater Setpoint Temperature {C}
    ,                        !- Basin Heater Operating Schedule Name
    ,                        !- Evaporation Loss Mode
    0.2,                     !- Evaporation Loss Factor {percent/K}
    0.008,                   !- Drift Loss Percent {percent}
    ,                        !- Blowdown Calculation Mode
    3,                       !- Blowdown Concentration Ratio
    ,                        !- Blowdown Makeup Water Usage Schedule Name
    ,                        !- Supply Water Storage Tank Name
    ,                        !- Outdoor Air Inlet Node Name
    FanCycling,              !- Capacity Control
    1,                       !- Number of Cells
    MinimalCell,             !- Cell Control
    0.33,                    !- Cell Minimum  Water Flow Rate Fraction
    2.5,                     !- Cell Maximum Water Flow Rate Fraction
    1;                       !- Sizing Factor
```

**Comment by macumber** [comment]
@kbenne FYI

**Comment by macumber** [comment]
E+ 8.6 file for reference

[old.idf.txt](https://github.com/NREL/EnergyPlus/files/842461/old.idf.txt)

**Comment by mjwitte** [comment]
This has been fixed in current develop.

**Comment by mjwitte** [comment]
Fix went in with #6012
